services/QuillLMS/app/models/teacher_activity_feed.rb

```diff
@@ -0,0 +1,78 @@
+# Usage: TeacherActivityFeed.add(teacher_id, activity_session_id)
+# Usage: TeacherActivityFeed.get(teacher_id)
+class TeacherActivityFeed < RedisFeed
+  KEY = 'teacher-'
```

Comment:
Might we want something more descriptive/less likely to accidentally collide here?